README.md

```diff
@@ -135,6 +135,28 @@ mcp = FastMCP("My App")
 
 # Specify dependencies for deployment and development
 mcp = FastMCP("My App", dependencies=["pandas", "numpy"])
+
+# Add lifespan support for startup/shutdown
+@asynccontextmanager
+async def app_lifespan(server: FastMCP) -> AsyncIterator[dict]:
```

Comment:
Is it possible for this to be typed more usefully than `dict`? So, e.g., accessing `request_context.lifespan_context` ends up well typed later on.

Reply:
Reading the code below, it seems like it is actually strongly typed, I guess? Using `dict` in the example might be a bit misleading. Maybe we could at least use a `TypedDict`?